### PR TITLE
Removing echo t > /proc/sysrq-trigger command from log collection

### PR DESCRIPTION
**Pull request by nikolaypopov**
Removing echo t > /proc/sysrq-trigger command from log collection, because it freezes instances and causes timeouts.

Signed-off-by: nikolaypopov <nikolay.popov86@gmail.com>

### CODE REVIEW

**Comment by vilasd** [comment]
Put a comment on top the line to add the echo t later wherever necessary.

File: tests/common.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ func CollectSupport() {
 			expect(nodes).NotTo(beEmpty())
 
 			journalCmd := fmt.Sprintf(
-				"echo t > /proc/sysrq-trigger && journalctl -l > ~/all_journal_%v",
+				"journalctl -l > ~/all_journal_%v",
```
Comment (harsh-px) [inline tests/common.go]: remove the journal too.. not needed.

Comment (nikolaypopov) [reply tests/common.go]: @harsh-px If we remove journalctl command as well there is no need for this function at all then, should we just disable this func?

Comment (nikolaypopov) [reply tests/common.go]: Removed CollectSupport()

**Pull request merged**